src/Fantomas.Core.Tests/StructTests.fs

```diff
@@ -127,3 +127,26 @@ namespace meh
 
 type TupleStruct = (struct (string * string))
 """
+
+[<Test>]
+let ``struct empty type`` () =
+    formatSourceString
+        false
+        """
+type NameStruct() =
```

Comment:
Is it ok to add additional tests? Because there could be changes in the future which break the empty line behavior again which wouldn't be caught by current tests

Reply:
I wouldn't mind, but add a comment that the expected result is not entirely correct.

If the input is:

```fsharp
type X =
  
    interface
    // comment
    end
```

that exact code should be produced. Which will not be the case with the current changes.
If somebody later ever breaks that test, by having a better result, then that person will at least know what is up.

Reply:
I don't want to go as far as adding tests for broken functionality. I just wanted to add tests for edge cases identified during this PR. See my latest changes :)